src/worker.cc

```diff
@@ -961,6 +961,32 @@ void ParseApi(const std::string& request, Options::Action action, valhalla::Api&
   from_json(document, *api.mutable_options());
 }
 
+std::string jsonify_error(const valhalla_exception_t& exception, const Api& request) {
```

Comment:
this was pulled out of the service only method below and is now used when the actor methods throw, which is what the workers do. we could move the try catch inside the actor and have it handle it all but taht would require rewriting some unit tests and i didn twant to bite that off yet. we can leave that for the future